setup.py

```diff
@@ -59,7 +68,10 @@ def finalize_options(self):
     def run(self):  # noqa
         """Run command to remove temporary files and directories."""
         os.chdir(my_dir)
-        os.system('rm -vrf ./build ./dist ./*.pyc ./*.tgz ./*.egg-info')
+        os.system(
+            'rm -vrf ./build ./dist ./*.pyc ./*.tgz ./*.egg-info '
+            + './docker-context-files/*.whl ./docker-context-files/*.tgz'  # noqa
+        )
```

Comment:
```suggestion
        os.system(
            'rm -vrf ./build ./dist ./*.pyc ./*.tgz ./*.egg-info '
            './docker-context-files/*.whl ./docker-context-files/*.tgz'
        )
```

This should work right? and we can remove `noqa` ?

Reply:
Nope. # noqa was there for another reason (security of os.system for user input) which was a false-positive, But I can double-check,